### PR TITLE
Update postBuild to use or new nightly pyopenms name scheme

### PR DESCRIPTION
**Pull request by timosachsenberg**

### CODE REVIEW

**Comment by jpfeuffer** [comment]
Did you also check the environments and requirements files?

**Comment by timosachsenberg** [comment]
only environments (which doesn't have pyopenms in it). Will update requirements

**Comment by timosachsenberg** [comment]
I hope this still selects our nightlies

**Comment by jpfeuffer** [comment]
The test code action should catch that. It will take the newest one that is on our server which should always be a nightly. If you want a release (e.g., for a release branch of the docs) you can put a version selector there. That's the idea.

**Comment by timosachsenberg** [comment]
ok sounds good. can we document this somehow?

File: postBuild [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-python3 -m pip install --index-url https://pypi.cs.uni-tuebingen.de/simple/ pyopenms-nightly
+python3 -m pip install --index-url https://pypi.cs.uni-tuebingen.de/simple/ pyopenms
```
Comment (jpfeuffer) [inline postBuild:3]: Is this even necessary? I thought binder installs based on the environment/requirements?

Comment (timosachsenberg) [reply postBuild:3]: no clue... maybe related to caching the base environment without postBuild stuff?

**Pull request merged**